libs/coin-evm/src/api/explorer/types.ts

```diff
@@ -16,7 +17,7 @@ export type ExplorerApi = {
   }>;
 };
 
-type ExplorerConfig = EthereumLikeInfo["explorer"];
+type ExplorerConfig = EvmConfig["explorer"];
```

Comment:
Not sure you need to change this. What we want is to be sure that the `ExplorerConfig` is the kind describe in `EthereumLikeInfo`

Reply:
Actually the EthereumLikeInfo object only has `{ chainId: number}` so `explorer` and `node` are only information that can be found in the config